examples/experiment_eval.py

```diff
@@ -147,7 +148,7 @@ def _iterate_env_agent_combo_not_in_db(
                 terminal_evaluators=[
                     ReachGoalLLMEvaluator(
                         model_names["env"],
-                        EvaluationForTwoAgents[SotopiaDimensions],
+                        EvaluationForTwoAgents[EvaluationByGroup],
```

Comment:
This `EvaluationByGroup` should only be applied once? @yeonir @bugsz Any chance we can do a class like `EvaluationForGroup` to replace the `EvaluationForTwoAgents`?

Reply:
I think in this case we should use `TeamEvaluationDimensions` (something like this) as the name?
`EvaluationForGroup` needs include evaluations for multiple (>=3) agents?